python/lsst/sconsUtils/state.py

```diff
@@ -172,14 +172,20 @@ def _initEnvironment():
     if env['PLATFORM'] == 'darwin':
         env['LDMODULESUFFIX'] = ".so"
         if not re.search(r"-install_name", str(env['SHLINKFLAGS'])):
-            env.Append(SHLINKFLAGS=["-Wl,-install_name", "-Wl,${TARGET.file}"])
+            env.Append(SHLINKFLAGS=["-install_name", "@rpath/${TARGET.file}"])
```

Comment:
PS I'm not going to fight this. I'm okay with adjusting the patch so it just cleans up the `-Wl` stuff. I'll see what @jdswinbank says when he reviews it.